File: programmers/올바른_괄호.js
function solution(s){
    let check = []
    
    for(let i=0; i<s.length; i++){
        if(s[i] === "("){
            check.push("(")
        }else if(!check.length){
            return false
        }else{
            check.pop()
        }
    }

    return !check.length ? true : false
}

File: programmers/올바른_괄호.py
def solution(s):
    check = []

    for i in range(len(s)):
        if s[i] == "(":
            check.append("(")
        
        elif (len(check) == 0):
            return False
        
        else :
            check.pop()
    
    return True if len(check)==0 else False
